llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -2238,6 +2238,9 @@ X86TargetLowering::X86TargetLowering(const X86TargetMachine &TM,
     }
   }
 
+  if (!Subtarget.useSoftFloat() && Subtarget.hasAVX())
+    setOperationAction(ISD::FABS, MVT::v8f16, Custom);
```

Comment:
Please can you move this inside the main "Subtarget.hasAVX()" section, around line 1425?